src/osp/Active/SysAreaAssociate.cpp

```diff
@@ -199,8 +199,7 @@ void SysAreaAssociate::floating_origin_translate(Vector3 translation)
 }
 
 
-int SysAreaAssociate::sat_activate(universe::Satellite sat,
-                                   universe::UCompActivatable &satAct)
+int SysAreaAssociate::sat_activate(universe::Satellite sat)
```

Comment:
I think you said that Satellite, in this case, is just an identifier.

Maybe we should be naming the types that are just handles / IDs into the registry with names that indicate they are such.

E.g. SatelliteHandle_t, SatelliteId_t

Reply:
An ECS entity is kind of its own special thing. It's just an id, but it's treated like a container for components. Maybe SatelliteEnt, or UniverseEnt in the same way ActiveEnt is named?

Reply:
That's an easy enough naming convention to teach someone, so i'd be fine with it.